src/utils.js

```diff
@@ -366,26 +385,25 @@ function magnitude(arr) {
     return Math.sqrt(arr.reduce((acc, val) => acc + val * val, 0));
 }
 
-
 class Callable extends Function {
     constructor() {
-        let closure = function (...args) { return closure._call(...args) }
-        return Object.setPrototypeOf(closure, new.target.prototype)
+        let closure = function (...args) {
+            return closure._call(...args);
+        };
+        return Object.setPrototypeOf(closure, new.target.prototype);
     }
 
-    _call(...args) {
-        throw Error('Must implement _call method in subclass')
+    _call() {
```

Comment:
Feels strange to just delete arguments, for documentation and it also has practical effects:

![image](https://user-images.githubusercontent.com/5236548/227589289-90ebac40-51dd-47da-83a6-da99b928f7d7.png)